templates/repo/diff/options_dropdown.tmpl

```diff
@@ -1,7 +1,7 @@
 <div class="ui dropdown tiny basic button icon-button tooltip" data-content="{{.locale.Tr "repo.diff.options_button"}}">
 	{{svg "octicon-kebab-horizontal"}}
 	<div class="menu">
-		<a class="item tiny basic toggle button" data-target="#diff-files">{{.locale.Tr "repo.diff.show_diff_stats"}}</a>
+		<a class="item tiny basic toggle button" onclick="window.config.pageData.diffFileInfo.fileListIsVisible = !window.config.pageData.diffFileInfo.fileListIsVisible;">{{.locale.Tr "repo.diff.show_diff_stats"}}</a>
```

Comment:
I am not sure whether it's good to use this inline script. IIRC some maintainers proposed to avoid inline scripts as much as possible. And it's not clear that it triggers the Vue's state Reactivity

Reply:
Removed the inline js code.